{{cookiecutter.project_name}}/docs/02_notebooks/library_example.ipynb

```diff
@@ -10,8 +10,7 @@
     "In this library template, notebooks will be executed with pytest (thus on every\n",
     "commit in your CI/CD pipeline). The results of the executions will be saved to the docs directory and converted to\n",
     "static websites through nbconvert. The static websites are then added to the documentation under the\n",
-    "_Guides and Tutorials_ section. These websites will be deployed to gitlab/github pages on push to develop.\n",
-    "For azure-devops, the CI pipeline currently does not include a deployment of the pages - pull requests are welcome :)"
+    "_Guides and Tutorials_ section. These websites will be deployed to Github pages on push to develop."
```

Comment:
The output of one of the functions mentions pylint, pls change that text

Also, pls demonstrate how to use references in the nb (like in tianshou)

Reply:
@carlocagnetta I did the first part, could you add the references pls?

Reply:
Merging this meanwhile, you can go ahead and use this for the new project!